### ISSUE TITLE
Windows xp setup does not detect floppy disk created by packer

### ISSUE THREAD

**Issue by eiximenis** [mitchellh/packer#2421]
Standard way of create unattended setups of Windows XP is having a floppy disk with a winnt.sif file.
But the windows xp setup does not detect the floppy disk created by packer, so the unattended setup is not possible this way.

A disk image created by dd and mkfs.msdos (more info in http://fatslowkid.blogspot.com.es/2010/09/installing-windows-xp-unattended-with.html) is well detected by Windows XP setup.

Comparing both binary files some differences arise between the packer disk image and the disk image created by dd:
1. Sectors are marked using header 0xf8ff (FAT16) instead 0xf0ff (FAT12)
2. Media descriptor type of the packer image is 0xf8 (HDD) instead of 0xf0 (floppy disk 1.4MB)
3. Num of root directory entries is 0x200 (512 recommended FAT16 value) instead 0xe0 (FAT12 value)
4. Num of sectors per FAT is 0xc instead 0x9
5. Num sectors per track is 0x20 instead 0x12
6. Num of heads is 0x10 instead 0x2

Also sectors seems to start in a physical different location in disk image created by packer than the image created by dd / mkfs.msdos.

Need to take a look but seems that packer is generating a FAT16 image of the floppy disk and this is not recognized by the setup of windows xp.
It is just the setup: once windows xp is installed the floppy disk is recognized.

Greetings!
edu

**Comment by icnocop** [comment]
It seems that the same thing happens with Windows Server 2003.

To test, I pressed F6 during setup initialization, to install a third party SCSI or RAID driver.
I then pressed S to specify additional SCSI adapters, CD-ROM drives, or special disk controllers for use with Windows, including those for which you have a device support disk from a mass storage device manufacturer.

The following error message was displayed by Windows Server 2003 (64-bit) setup:
> File txtsetup.oem caused an unexpected error (32768) at line 1896 in d:\srvrtm\base\boot\setup\oemdisk.c.
